src/lib/remark-plugins/rewrite-static-tutorials-assets/index.ts

```diff
@@ -25,13 +25,23 @@ import { Image, Definition } from 'mdast'
 const ASSET_API_ENDPOINT =
 	process.env.ASSET_API_ENDPOINT || `${process.env.MKTG_CONTENT_API}/api/assets`
 
+/**
+ * This Plugin rewrites src asset paths for tutorials content. With tutorials, images live in a
+ * /public directory in the tutorials repository.
+ *
+ * For dev portal previews / prod, we source these image paths from the mktg-content-api,
+ * which uses the GitHub API. For tutorials repo previews, we use the PREVIEW_BRANCH env
+ * to target the correct path via the GitHub API.
+ *
+ * For authors working on content locally, we spin up a custom asset server within docker
+ * and the paths are served 1-1.
+ */
```

Comment:
Just added more context here on why this plugin exists. The local tutorials workflow can be confusing.

Reply:
Great call here!

💭 It probably wouldn't hurt to include a link to the Content API endpoint in this comment, mostly for easy click-through for _us engineers_ 🤔 

Something like:
> See: https://github.com/hashicorp/mktg-content-workflows/blob/main/api/assets.ts

(And the repository is private so I think this is OK.)

Reply:
Good idea!